### PR TITLE
Modify daily to static

### LINKED ISSUE #81: Modify dailyMedicationManager to static

### PR DESCRIPTION
**Pull request by T0nyLin**
# Checklist for feature implementation and modification
Closes #81
- [X] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [x] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #81
- [ ] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [ ] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information

Nil

### CODE REVIEW

**Review by annoy-o-mus — CHANGES_REQUESTED** [review]
Great job in making the code simpler to read and process! Just a couple of document change requests and code suggestions

File: src/main/java/meditracker/DailyMedicationManager.java [hunk 1]
```diff
@@ -14,16 +14,15 @@
  * @see DailyMedication
  */
 public class DailyMedicationManager {
-    private final List<DailyMedication> dailyMedications;
+    private static List<DailyMedication> dailyMedications = new ArrayList<>();
 
     /**
      * Constructs DailyMedicationManager with a list of DailyMedication
```
[the remark this answers is not in the document]
Comment (T0nyLin) [reply src/main/java/meditracker/DailyMedicationManager.java]: Thank you for pointing it out, updated JavaDoc in 1e45ea9

File: src/main/java/meditracker/DailyMedicationManager.java [hunk 2]
```diff
@@ -36,8 +35,7 @@ public DailyMedicationManager(MedicationManager medicationManager) {
      *
```
[the remark this answers is not in the document]
Comment (T0nyLin) [reply src/main/java/meditracker/DailyMedicationManager.java:36]: Thank you for pointing it out, updated JavaDoc in 1e45ea9

File: src/main/java/meditracker/DailyMedicationManager.java [hunk 3]
```diff
@@ -48,12 +46,16 @@ public DailyMedicationManager(List<String> lines) {
         }
     }
 
+    protected static void clearDailyMedication() {
+        dailyMedications = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (T0nyLin) [reply src/main/java/meditracker/DailyMedicationManager.java]: Yes, thx for the suggestion, implemented in 1e45ea9

**Review by annoy-o-mus — APPROVED** [review]
Looks good to go

**Pull request merged**